string storage_place_name
string drop_place_name